learn/on-premise/observability/logs-aggregation/kibana.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Kibana"
+id: ""
+sidebar_label: "Kibana"
+---
+---
+
+### Introduction
+
+It's is considered as a good practice for a *Backend Developer* to add log statements in his code. So, when the application is deployed he can go through the
```

Comment:
We should clarify that this is of interest only to WaveMaker team. Logs of the applications developed on WaveMaker do not get sent here.